docs/docs/index.md

```diff
@@ -103,3 +103,17 @@ kafka-console-consumer \
 --topic application.communication.messages \
 --from-beginning
 ```
+
+## Known Issues
```

Comment:
I'd have this in the test doc... definitely not so important to be on the home page